contrib/endpoints/src/api_manager/config_test.cc

```diff
@@ -93,13 +93,13 @@ TEST(Config, ServerConfigProto) {
 }
 
 static const char kInvalidServerConfig[] = R"(
-service_control_config {
-  type: 1
-  config {
-    cache_entries: 1020
-    flush_interval_ms: 15
+  service_control_config {
+    type: 1
```

Comment:
The safest thing is to not have any leading or trailing white spaces. That way the strings will work with all parsers.